packages/neon-core/src/sc/ContractParam.ts

```diff
@@ -16,22 +16,44 @@ export enum ContractParamType {
   Signature = 0x17, // TODO: Implement support
 
   Array = 0x20,
-  Map = 0x22, // TODO: Implement support
+  Map = 0x22,
 
   InteropInterface = 0x30, // TODO: Implement support
 
   Void = 0xff,
 }
 
+export type ContractParamMap = {

```

Comment:
Propose to rename this to `ContractParamMapEntry`.  `ContractParamMap` is too close to the type `ContractParam.Map` and causes some confusion.